### PR TITLE
Launch Manage Special Access Dialog and Handle Save/Cancel/Close scen…

### PR DESCRIPTION
**Pull request by wongvincent**
…arios

https://rally1.rallydev.com/#/29180338367d/detail/userstory/397388004180/tasks

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-special-access-editor.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import '@brightspace-ui/core/components/colors/colors.js';
 import '@brightspace-ui/core/components/icons/icon.js';
 import '@brightspace-ui/core/components/button/button-subtle.js';
+import '@brightspace-ui/core/components/button/floating-buttons.js';
```
Comment (mdgbayly) [inline components/d2l-activity-editor/d2l-activity-special-access-editor.js:3]: is this being used? Is it somehow used by the LMS dialog?

File: components/d2l-activity-editor/d2l-activity-special-access-editor.js [hunk 2]
```diff
@@ -78,11 +78,67 @@ class ActivitySpecialAccessEditor extends ActivityEditorMixin(RtlMixin(LocalizeM
...
+		];
+
+		// Launch into our "friend", the LMS, to do the thing.
+		const delayedResult = D2L.LP.Web.UI.Legacy.MasterPages.Dialog.Open(
+			/*               opener: */ document.body,
+			/*             location: */ location,
+			/*          srcCallback: */ 'SrcCallback',
+			/*       resizeCallback: */ '',
+			/*      responseDataKey: */ 'result',
+			/*                width: */ 1920,
```
Comment (mdgbayly) [inline components/d2l-activity-editor/d2l-activity-special-access-editor.js:120]: Would be awesome if just setting a really big width/height forced the dialog into a mobile rendering but I guess that is too much to expect?

File: components/d2l-activity-editor/d2l-activity-special-access-editor.js (hunk 1 above)
Comment (wongvincent) [reply components/d2l-activity-editor/d2l-activity-special-access-editor.js]: sorry, that was leftover from when I was experimenting with the simple-overlay and adding custom floating buttons to that!

File: components/d2l-activity-editor/d2l-activity-special-access-editor.js (hunk 2 above)
Comment (wongvincent) [reply components/d2l-activity-editor/d2l-activity-special-access-editor.js:119]: that's too much to ask! 😆 
we'll figure something out when we get to the fullscreen dialog attribute and styling!

**Pull request merged**